### PR TITLE
fix: revert `DatePicker`, update `Tile` story, revert theme order

### PR DESCRIPTION
**Pull request by SimonFinney**
Resolves https://github.com/carbon-design-system/ibm-security/pull/241#pullrequestreview-367401980

## Proposed changes

- Revert required `DatePicker` overrides
- Revert global theme ordering to ensure theme tokens are configured
- Update `Tile` story to reflect Carbon

### CODE REVIEW

File: src/components/DatePicker/_index.scss [hunk 1]
```diff
@@ -1,7 +1,87 @@
...
 /// Date picker component.
 /// @group date-picker
-/// @copyright IBM Security 2019
+/// @copyright IBM Security 2018 - 2020
 ////
 
 @import 'carbon-components/scss/components/date-picker/date-picker';
+
+@include export-namespace($name: datepicker) {
+  // TODO - V2: Colors need to be resolved on Carbon's end - they need to add grey-100 support for this component
```
Comment (jendowns) [inline src/components/DatePicker/_index.scss]:
```
  // TODO - V2: Colors need to be resolved on Carbon's end - they need to add gray100 support for this component
```

**Comment by SimonFinney** [comment]
@jendowns Thanks - I've updated the title and description accordingly

**Comment by jendowns** [comment]
Thanks @SimonFinney -- I see there are (seemingly?) unrelated changes to two components here though. 😅 I'm just thinking of the change log here, and what makes the most sense. I understand that grouping things is easier but splitting unrelated changes is also super helpful when the history is reviewed. Seems like a good process as well. Do you agree?

**Comment by SimonFinney** [comment]
Sorry, I half-read this - Let me break it up! Thanks

**Pull request closed without merging**